### PR TITLE
Update @supabase/postgres-meta in studio

### PR DESCRIPTION
**Pull request by ruggi99**
## What kind of change does this PR introduce?

Chore: Upgrade @supabase/postgres-meta to `v0.60.0`

Also resolves this annoying warning:
```sh
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@supabase/postgres-meta@0.26.1',
npm WARN EBADENGINE   required: { node: '>=14 <15', npm: '>=7' },
npm WARN EBADENGINE   current: { node: 'v18.13.0', npm: '8.19.3' }
npm WARN EBADENGINE }
```

Had to downgrade `pgsql-parser` to `v13.4.0` because dependency `libpg-query` last version pre-built is `v13.3.0` on `https://supabase-public-artifacts-bucket.s3.amazonaws.com/` at the moment.

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: studio/components/interfaces/Database/Tables/ColumnList.tsx
Comment (ruggi99) [inline studio/components/interfaces/Database/Tables/ColumnList.tsx:30]: Don't know if it's the correct pattern to follow

File: studio/components/interfaces/TableGridEditor/SidePanelEditor/ForeignKeySelector/ForeignKeySelector.tsx
Comment (ruggi99) [inline studio/components/interfaces/TableGridEditor/SidePanelEditor/ForeignKeySelector/ForeignKeySelector.tsx:74]: Should we add checks for undefined values?

File: studio/components/interfaces/TableGridEditor/SidePanelEditor/RowEditor/RowEditor.utils.ts
Comment (ruggi99) [inline studio/components/interfaces/TableGridEditor/SidePanelEditor/RowEditor/RowEditor.utils.ts:18]: Same as above

File: studio/lib/helpers.ts
Comment (ruggi99) [inline studio/lib/helpers.ts:2]: Unused import

File: studio/components/interfaces/TableGridEditor/TableGridEditor.tsx
Comment (ruggi99) [inline studio/components/interfaces/TableGridEditor/TableGridEditor.tsx:95]: Same as above

**Comment by ruggi99** [comment]
PR blocked until supabase/postgres-meta#481 merged or relative issue fixed.

File: studio/components/interfaces/Database/Tables/ColumnList.tsx
Comment (sweatybridge) [reply studio/components/interfaces/Database/Tables/ColumnList.tsx]: Using `!` will crash at runtime if the columns is actually undefined. I've updated the undefined check to use `?`.

**Review by sweatybridge — APPROVED** [review]

**Comment by ruggi99** [comment]
@sweatybridge please mind that overrides do not work very well in monorepos.
That's why I manually updated `package-lock.json`

**Comment by sweatybridge** [comment]
I found this issue https://github.com/npm/cli/issues/4232 which suggested the following fixes:

1. place `overrides` in root package.json
2. uninstall the dependency first before saving

```bash
npm uninstall @supabase/postgres-meta
npm i --save-dev @supabase/postgres-meta@^0.60.2
```

This made the package-lock.json honour the overrides properly.

**Comment by ruggi99** [comment]
Hope it works, npm is very tricky

**Comment by sweatybridge** [comment]
Yea, we should be updating the artefacts bucket soon anyway.

**Comment by ruggi99** [comment]
@sweatybridge 
Lines added to `Dockerfile` were there because it fails to build on `arm64`.
Next time you build docker studio image it should fail.
At least you know what is the fix to apply in case.

**Comment by sweatybridge** [comment]
I see. Let me check arm64 build again. It was working for x86 so I assumed it was ok. Let me add it back

**Comment by ruggi99** [comment]
Yeah, this is due public artifacts never available for `arm64`, that's why it was working on `x86`

**Pull request merged**
